dcicutils/beanstalk_utils.py

```diff
@@ -393,9 +393,9 @@ def _get_beanstalk_configuration_settings(env):
             }
     """
     try:
-        client = boto3.client('elasticbeanstalk')
+        client = boto3.client('elasticbeanstalk', region_name='us-east-1')
         config = client.describe_configuration_settings(ApplicationName='4dn-web', EnvironmentName=env)
-        options = config['ConfigurationSettings']['OptionSettings']  # guaranteed to be present
+        options = config['ConfigurationSettings'][0]['OptionSettings']  # guaranteed to be present
```

Comment:
If it's the only one, you could also do this as (untested):
```suggestion
        # These are guaranteed to be present
        [settings] = config['ConfigurationSettings']
        options = settings['OptionSettings']
```
but what you have is fine.